### DISCUSSION TITLE
现在有gvs 和gws 的ip，在tls模式下各位有什么新策略没？

### DISCUSSION THREAD

**Discussion by wpyok500** [SeaHOH/GotoX#57] [一般]
现在有gvs 和gws 的ip，在tls模式下各位有什么新策略没？

**Comment by SeaHOH** [comment]
IPv4 直连已经玩不转了，不管什么方式必须加跳板。IPv6 也许还能用一阵。

没用 QUIC，不清楚情况。

**Comment by wpyok500** [comment]
qjic ipv6  sni  php 我均有用，我主要用还是quic

**Comment by wpyok500** [comment]
目前我知道两个ip可以直接使用tls协议，看了下证书是edgestatic.com，主要是没找到这样的ip段来扫。

**Comment by SeaHOH** [comment]
如果这个 IP 属于谷歌或 GGC，那么就是还没封完，否则这很可能是端口转发。

**Comment by blob2015** [comment]
ipv6也是会很快奄奄一息，自建大概是比较靠谱的一个方法，但是某种意义上来说，如果那天金融信息口进一步严控【今年已经把hk的离岸差不多给废了】，真要发力，更加容易打击——购买需要通过金融口
只能是见招拆招了，没啥绝对的安逸
quic目前还能继续苟延残喘一段时间，但是不时需要重启下路由器了
大抵会成为玩具，一旦公开、流行，就很容易被杀

**Comment by wpyok500** [comment]
xx不是新内置一些，tls协议的ip吗

**Comment by blob2015** [comment]
@Just2co 
感谢老兄技术信息分享。又想起以前的那些个海量issue，各种懂技术的，可以给俺们这些小白偶尔说说。
至于汇率、金融，个人觉得，目前世界就没有一个安逸的地方……屁股都有屎，多些、少些的差别……轻浮、贪婪、疯狂，好像一个都逃不了。
关于金融、汇率，网络流行的论述，大概可以分为两方，一方乐观，一方悲观，但大多感情色彩强烈，逻辑链往往不严谨、甚至极为不严谨。个人小虾米一只，难以说太多、判断太多，偶尔翻翻他们的片言只语，也就仅此而已……嗯，其中一方有时寒冰（本来还好，但个人感觉近年，其内郁的情结，愈发……）之类的，另外一方有诸如国师（唉）之类的……
